extensions/hibernate-search-elasticsearch/runtime/src/main/java/io/quarkus/hibernate/search/elasticsearch/runtime/HibernateSearchElasticsearchRuntimeConfig.java

```diff
@@ -128,9 +128,18 @@
         /**
          * The default configuration for the Elasticsearch indexes.
          */
-        @ConfigItem
+        @ConfigItem(name = ConfigItem.PARENT)
         ElasticsearchIndexConfig indexDefaults;
 
+        /**
+         * @deprecated Define index defaults directly at the backend level instead of using {@code index-defaults};
+         *             e.g. use simply {@code quarkus.hibernate-search.elasticsearch.indexing.queue.count}
+         *             instead of {@code quarkus.hibernate-search.elasticsearch.index-defaults.indexing.queue.count}.
+         */
+        @ConfigItem(name = "index-defaults")
+        @Deprecated
+        ElasticsearchIndexConfig legacyIndexDefaults;
```

Comment:
Given we are still experimental, I would just drop it and document it in the release notes.

Reply:
Ok, I'll add a commit.